### PR TITLE
feat: Update the logo with the EddieHub Mascot

### LINKED ISSUE #265: [FEATURE] Update the logo with the EddieHub Mascot
### Description

https://github.com/EddieHubCommunity/Branding

Concerned file: https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/blob/main/src/images/logos/eddiehub.png

### Screenshots

_No response_

### Additional information

_No response_

### LINKED ISSUE #310: [FEATURE] Add Links to Partner Icons
### Description

It's better to have links to the partner icons.

- Wemakedevs https://wemakedevs.org/
- 4C https://www.4c.rocks/

### Screenshots

Screenshot from 2023-11-06 17-28-38

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ninadvyas**
`Closes #265 ` Update the logo with the EddieHub Mascot :  

## Changes proposed

- Change the previous logo in the header with the eddihub mascot.
- Change the file with the new logo accordingly & delete previous file.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots of #265 
Before:
Screenshot 2023-12-08 at 12 38 35 PM

After:
Screenshot 2023-12-08 at 12 39 16 PM

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ export function Footer() {
           >
             {[
               [
```
Comment (xkrishguptaa) [inline src/components/Footer.jsx:26]: These changes look irrelevant, please make them in a different PR

Comment (ninadvyas) [reply src/components/Footer.jsx:26]: @xkrishguptaa what exactly I need to do. As I am understanding I need to make another PR for the footer changes am right?

Comment (xkrishguptaa) [reply src/components/Footer.jsx:26]: @ninadvyas you are fixing two issues right? Please keep this PR only for the EddieHub Mascot fix and do the links of partner's fix in a different PR

Comment (ninadvyas) [reply src/components/Footer.jsx:26]: I am fixing two issues. Can you teach me how to make different PR because I have committed changes in my main branch and according to me I have to make another branch for another PR. Is that correct? @xkrishguptaa

**Comment by ninadvyas** [comment]
Hello @xkrishguptaa I have made changes this PR only changes the EddieHub Mascot fix.

**Review by adityaraute — APPROVED** [review]
LGTM

**Review by xkrishguptaa — APPROVED** [review]
